src/combinatorics/quiver.lean

```diff
@@ -3,130 +3,148 @@ Copyright (c) 2021 David Wärn. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: David Wärn
 -/
-import tactic
+import data.equiv.basic
+import order.well_founded
+import data.nat.basic
+import data.opposite
+
 /-!
 # Quivers
 
-This module defines quivers. A quiver `G` on a type `V` of vertices assigns to every
-pair `a b : V` of vertices a type `G.arrow a b` of arrows from `a` to `b`. This
+This module defines quivers. A quiver on a type `V` of vertices assigns to every
+pair `a b : V` of vertices a type `a ⟶ b` of arrows from `a` to `b`. This
```

Comment:
Perhaps we should add a brief note should be added about the `Sort` vs `Type` issue for arrows?

This is completely optional, as it seems that the intention is to (eventually) replace `Sort` with `Type`.